dataprep/eda/basic/compute.py

```diff
@@ -1477,12 +1554,260 @@ def calc_stats(
         "Total Size in Memory": memory_usage,
         "Average Record Size in Memory": memory_usage / dim[0],
     }
+    column_conti = []
+    for column in df.columns:
+        if is_dtype(detect_dtype(df[column]), Continuous()):
+            column_conti.append(column)
+
+    column_calced = []
+    ks_tests = []
+    for column1 in column_conti:
```

Comment:
Try to avoid using `in` operation for list (`in` is efficient for set). Here I think you can use the index to iterate. Something like:
```python
For i in range(len(column_conti))
  For j in range(i+1, len(column_conti)):
    ...
```